guillotina/contrib/memcached/driver.py

```diff
@@ -94,7 +94,12 @@ def __init__(self, operation: str):
 
 except ImportError:
     _SEND_METRICS = False
-    watch = metrics.watch  # type: ignore
+
+    class watch(metrics.watch):  # type: ignore
```

Comment:
oh! This might be a problem in other modules as well! Can you verify the error isn't present?

Reply:
I'll fix it in a separate PR